Team417_2021/src/main/java/org/firstinspires/ftc/team417_2021/ginisTeleOp.java

```diff
@@ -0,0 +1,142 @@
+package org.firstinspires.ftc.team417_2021;
+
+import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
+import com.qualcomm.robotcore.eventloop.opmode.TeleOp;
+import com.qualcomm.robotcore.hardware.CRServo;
+import com.qualcomm.robotcore.hardware.DcMotor;
+import com.qualcomm.robotcore.hardware.DcMotorSimple;
+import com.qualcomm.robotcore.hardware.Servo;
+
+import org.firstinspires.ftc.team417_2021.Resources.Toggler;
+
+@TeleOp(name = "Test TeleOp")
+public class ginisTeleOp extends LinearOpMode {
+
+    public Toggler wristToggle = new Toggler();
+    public CRServo wristServo;
+    public DcMotor motorFL;
+    public DcMotor motorBL;
+    public DcMotor motorFR;
+    public DcMotor motorBR;
+
+    boolean toggleState = true;
+
+    public DcMotor motorArm1;
+    public DcMotor motorArm2;
+
+    public DcMotor motorSlides;
+
+
+    @Override
+    public void runOpMode() throws InterruptedException {
+
+        motorFL = hardwareMap.dcMotor.get("motorFL");
+        motorBL = hardwareMap.dcMotor.get("motorBL");
+        motorFR = hardwareMap.dcMotor.get("motorFR");
+        motorBR = hardwareMap.dcMotor.get("motorBR");
+
+        wristServo = hardwareMap.crservo.get("wristServo");
+        wristServo.resetDeviceConfigurationForOpMode();
+        wristServo.setPower(0);
+
+        motorArm1 = hardwareMap.dcMotor.get("arm1");
+        motorArm2 = hardwareMap.dcMotor.get("arm2");
+
+        motorSlides = hardwareMap.dcMotor.get("motorSlides");
+
+        motorFL.setMode(DcMotor.RunMode.STOP_AND_RESET_ENCODER);
+        motorBL.setMode(DcMotor.RunMode.STOP_AND_RESET_ENCODER);
+        motorFR.setMode(DcMotor.RunMode.STOP_AND_RESET_ENCODER);
+        motorBR.setMode(DcMotor.RunMode.STOP_AND_RESET_ENCODER);
+        motorArm1.setMode(DcMotor.RunMode.STOP_AND_RESET_ENCODER);
+        motorArm2.setMode(DcMotor.RunMode.STOP_AND_RESET_ENCODER);
+        motorSlides.setMode(DcMotor.RunMode.STOP_AND_RESET_ENCODER);
+
+
+         //motorArm1.setZeroPowerBehavior(DcMotor.ZeroPowerBehavior.FLOAT);
+        motorArm1.setZeroPowerBehavior(DcMotor.ZeroPowerBehavior.BRAKE);
```

Comment:
No commented out code
```suggestion
        motorSlides.setMode(DcMotor.RunMode.STOP_AND_RESET_ENCODER);

        motorArm1.setZeroPowerBehavior(DcMotor.ZeroPowerBehavior.BRAKE);
```